### ISSUE TITLE
[low] JAWS+Chrome: Terms of Service: Empty heading announced

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5496]
## Action Performed:
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Press Tab key to focus the link 'terms of service' and press Space to activate it.
3. Once the page 'Terms of service opens, press H to navigate through the headings.
4. Note what headings and levels the page has.

## Expected Result:
No empty headings should be announced.

## Actual Result:
There is an empty heading announced as 'heading level 5'.

HTML:
`<h5>&nbsp;</h5>`

Other occurrences:
1. Licenses -> an empty heading level 5 is focused and announced.
2. Privacy policy -> the same empty heading level 5 is focused and announced.

## Workaround:
No workaround

## Area issue was found in:
Terms of service page, Licenses page, Privacy policy page

## Failed WCAG checkpoints
1.3.1

## User impact:
The screen reader user might not understand the structure of page.

## Suggested resolution:
It is a best practice to ensure that all heading elements (those marked with `<h1>` through `<h6>`) contain content. Furthermore, make sure that the heading text is not hidden from screen readers (for example, using CSS display: none or aria-hidden="true"). See: https://www.w3.org/WAI/tutorials/page-structure/headings/

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134707463-d81e3465-bb53-4406-a0e9-731462778a1f.mp4

![Bug5181053_Empty_Heading-Licenses](https://user-images.githubusercontent.com/88733897/134707473-7ef4a552-7488-4fa9-917e-37f74081a3c2.png)
![Bug5181053_Empty_heading-terms_of_service](https://user-images.githubusercontent.com/88733897/134707484-12151639-cdc2-48ea-adcd-b9e284387334.png)
![Empty_heading](https://user-images.githubusercontent.com/88733897/134707488-38bac7b2-5265-4d27-ae27-3f226f232c10.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
